full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -66,13 +86,455 @@ public void testWithNullUrlDir() {
         TestUtil.testFail(db, "CALL apoc.load.directory('*', null, {recursive: false}) YIELD value RETURN value", IllegalArgumentException.class);
     }
 
+    @After
+    public void clearDB() {
+        db.executeTransactionally("MATCH (n) DETACH DELETE n");
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void testAddListenerWithApocLoadDirectoryNotEnabled() {
+        apocConfig().setProperty(APOC_IMPORT_FILE_USE_NEO4J_CONFIG, true);
+        apocConfig().setProperty(APOC_LOAD_DIRECTORY_ENABLED, false);
+        try {
+            db.executeTransactionally("CALL apoc.load.directory.add('test','CREATE (n:Test)', '*.csv', '', {}) YIELD name RETURN name");
+        } catch (RuntimeException e) {
+            assertEquals("Failed to invoke procedure `apoc.load.directory.add`: Caused by: java.lang.RuntimeException: " + LoadDirectory.NOT_ENABLED_ERROR, e.getMessage());
+            throw e;
+        }
+    }
+
+    @Test
+    public void testAddMultipleFoldersAndRemoveAll() throws InterruptedException, IOException {
```

Comment:
per favore semplifica i test.
Facciamoli in questo modo:
* innanzitutto fallo in combinazione con la `apoc.load.csv` e `apoc.load.json`
* fai un test per `listenEventType`
* fai un test per subdirectory